### DISCUSSION TITLE
想吃青花鱼了，喵~

### DISCUSSION THREAD

**Discussion by LT-UT** [share121/inter-knot#3104] [其他]
![e4bf5d264869390bb2ed05ca04d30848661955685](https://github.com/user-attachments/assets/5adf23dc-f3c5-40e2-bb02-592fbfeda5f5)
求投喂

**Comment by seelesee** [comment]
吃，多吃qhy<img src="https://img-static.mihoyo.com/communityweb/upload/bf480233be8cd5436e79c93e6bcb0351.png" alt="青花卒" width="75" >

**Reply by LT-UT** [reply]
这鱼味道还可以，喵
但不是青花鱼，喵
这才是青花鱼，喵
![300](https://github.com/user-attachments/assets/c61078d6-ea7f-4100-b87d-f09b15e2498a)

**Reply by seelesee** [reply]
可以把崩崩qhy也吃了嘛

**Reply by LT-UT** [reply]
崩崩是什么？喵

**Reply by 00zzxx** [reply]
> 崩崩是什么？喵

应该是在说崩二策划吧(。-ω-)

**Comment by azb-666** [comment]
牢财我们想你了

**Comment by warning-Kirov-reporting** [comment]
小猫你可以吃青花鱼<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
